RiotSharp/StaticDataEndpoint/DataRequested.cs

```diff
@@ -5,7 +5,11 @@
     /// </summary>
     public enum ChampionData
     {
+        /// <summary>
+        /// The default. Resolves to type, version, data, id, key, name, and title
+        /// </summary>
         none,
+
```

Comment:
adding this one extra line to separate the default from the rest of them. Just thought it made more _visual_ sense. LMK if it works :)